q2_chemistree/plugin_setup.py

```diff
@@ -144,14 +144,19 @@
     name='Create a molecular tree',
     description='Build a phylogeny based on molecular substructures',
     inputs={'collated_fingerprints': FeatureTable[Frequency]},
-    parameters={'prob_threshold': Float % Range(0, 1, inclusive_end=True)},
+    parameters={'prob_threshold': Float % Range(0, 1, inclusive_end=True),
+                'distance_metric': Str},
```

Comment:
The type for this parameter should be `Choice`. For an example see here:
https://github.com/qiime2/q2-diversity/blob/master/q2_diversity/plugin_setup.py#L50